cpp/tests/io/json_test.cpp

```diff
@@ -1496,7 +1495,7 @@ TEST_P(JsonReaderParamTest, JsonDtypeParsing)
 
   auto int_col = int_wrapper{
     {0, 0, int_NA, 1, 1, int_NA, int_NA, int_NA, int_NA, 1, 0, int_NA, 1, 0, int_NA, int_NA},
```

Comment:
int_NA, double_NA and bool_NA don't seem relevant any more, since all those elements will be null in the output table

Reply:
That's right. I have to give with some data there. `int_NA` is just a placeholder data now.

Reply:
renamed to *_ignore.

Reply:
I would also be fine with zeroes, but maybe I', missing something in the test.